src/Bicep.LangServer/Completions/BicepCompletionProvider.cs

```diff
@@ -72,7 +73,9 @@ public IEnumerable<CompletionItem> GetFilteredCompletions(Compilation compilatio
                 .Concat(GetOutputValueCompletions(model, context))
                 .Concat(GetTargetScopeCompletions(model, context))
                 .Concat(GetImportCompletions(model, context))
-                .Concat(GetFunctionParamCompletions(model, context));
+                .Concat(GetFunctionParamCompletions(model, context))
+                .Concat(GetDisableNextLineDiagnosticsDirectiveCompletion(context))
+                .Concat(GetDisableNextLineDiagnosticsDirectiveCodesCompletion(model, context)); ;
```

Comment:
[nit] extra semicolon at the end here.

Reply:
Removed here:
https://github.com/Azure/bicep/commit/68d149805333288c691fd5ff7b71bbd80eadd731
Thanks!